src/main/java/com/rackspace/salus/monitor_management/services/ZoneManagement.java

```diff
@@ -329,5 +349,6 @@ public void removeAllTenantZones(String tenantId, boolean sendEvents) {
       }else {
         zoneRepository.deleteAllByTenantId(tenantId);
       }
+      zoneManagementSuccess.tags("operation","removeAll","objectType","zone").register(meterRegistry).increment();
```

Comment:
To align with others
```suggestion
      zoneManagementSuccess.tags("operation","removeAll","objectType","privateZone").register(meterRegistry).increment();
```